app.json

```diff
@@ -104,6 +104,12 @@
       "description": "API key for accessing Google services",
       "required": true
     },
+    "HEROKU_APP_NAME": {
+      "required": true
+    },
+    "HEROKU_PARENT_APP_NAME": {
+      "required": true
+    },
```

Comment:
It's not obvious to me what these settings should contain. Can we add a description? 

Does this mean we'll have to add these settings to all our deployments?

Reply:
They are special heroku environment variables which are populated by heroku with the name of the app and the name of the app's parent (if it's a PR build). They are marked as required but since they're filled in automatically there shouldn't be any extra work on our part. I'll add a description